_events/alumni-reunion.md

```diff
@@ -14,7 +14,8 @@ Thanks for checking in! The next Alumni Reunion will be this year starting Septe
 
 We can't wait to have you! If you have further questions please contact [Joe Halford](mailto:jahalfor@mtu.edu).
 
-Make sure to update your Alumni information on the [update form](http://infoupdate.mtutriangle.org/) so you receive information about the reunion as well as present day Active and Alumni activities.
+## Important!
```

Comment:
Can we have something a bit less aggressive and more positive, maybe something like "Keeping in touch". Same thing goes for line 18.